src/services/accomodation.service.js

```diff
@@ -0,0 +1,119 @@
+import db from '../database/models/index.js';
+const accomodations = db['accomodation']
+import { fileUpload } from "../helpers/multer";
+
+export const createAccomodation = async (req, res) => {
+    try {
+
+		if (req.file) {
+			req.body.image = await fileUpload(req);
+		} 
+		else {
+			req.body.image =
+				"https://images.unsplash.com/photo-1553095066-5014bc7b7f2d?ixlib=rb-1.2.1&ixid=MnwxMjA3fDB8MHxzZWFyY2h8Mnx8d2FsbCUyMGJhY2tncm91bmR8ZW58MHx8MHx8&w=1000&q=80";
+		}
+
+		const newUser = await accomodations.create(req.body);
+		const {name,description,location,image,geoLocation,highlight,amenities} = req.body;
+		return res.status(201).json({status:"success", data:{name,description,location,image,geoLocation,highlight,amenities}, message:"New Accomodation have been created"});
+	} catch (error) {
+		return res.status(500).json(error.message);
+	}
+};
+export const  getAllAccomodation = async (req, res) => {
+	try{
+        accomodations.findAndCountAll().then(accomodations => {
+				return res.status(200).json({status:"success",data:accomodations,message:"Retrieved"});
+			})	
+	}
+	catch(error) {
+		return res.status(500).json(error.message);
+	}
+};
+
+export const updateAccomodation = async(req, res) => {
+
+	try{
+	const id = req.params.id;
+	if (req.file) {
+		req.body.image = await fileUpload(req);
+	} 
+	else {
+		req.body.image =
```

Comment:
why re-assign image a value while you did it when creating accommodation

Reply:
I did this because in the database we have to enter something(it doesn't take a null value) in the image column and that is why I assign that image in case of uploading fails it takes that given image.